pyproject.toml

```diff
@@ -1,66 +1,60 @@
-[build-system]
-requires = ["setuptools==80.9.0"]
-build-backend = "setuptools.build_meta"
+[dependency-groups]
+dev = [
+  "cogapp",
+  "django-stubs",
+  "mypy",
+  "readable-log-formatter",
+]
 
 [project]
-name = "django-layout"
-version = "4.0"
-requires-python = ">=3.12"
 dependencies = [
-  "django==5.2.8",
   "dj-database-url",
-  "goodconf[yaml]",
   "django-alive",
   "django-csp",
+  "django==5.2.8",
```

Comment:
Should we do `5.2.*` here? I would want minor upgrades to be applied automatically.

Reply:
This diff is just noise from reordering. Renovate will automatically manage this; I have it configured to pin exact versions at the moment. This would always be set to the latest version. Could do it differently though!